sum_squares_of_first_n_natural_numbers.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+    int n=0;
+    cout<<"Input the value for 'n': ";  //Taking input for the value of 'n'
+    cin>>n;
+    int sum=0;
+    for(int i=1;i<n+1;i++)
+    {
+        sum = (n*(n+1)*((2*n)+1))/6;    //Mathematical Logic for calculating sum of squares of first n natural numbers.
+    }
```

Comment:
if n = 5.
sum = (5 * 6 * 11) / 6=55. (using formula)
you are repeating the formula for all values of n, whereas only one iteration is needed. 
Check once.